### ISSUE TITLE
What happen if the submitted strategy is over the max training time?

### ISSUE THREAD

**Issue by thkimYonsei** [ContinualAI/clvision-challenge-2024#12]
In this challenge, we submit our strategy and you evaluate them locally. 

If the strategy doesn't violate the restrictions (gpu memory usage and max training time) from our machine, but does violate the restriction of the challenge in your local machine, what will going to happen?

Do we get a chance to change some hyperparameters (e.g. training epochs or batch size) to fit into the restriction in your local machine? 

Thank you.

**Comment by mmasana** [comment]
Yes, we will talk with you and figure out if, by changing any hyperparameter, we can bring down the training time.
